spec/toolbar.spec.js

```diff
@@ -31,6 +31,19 @@ describe('MediumEditor.extensions.toolbar TestCase', function () {
             expect(document.querySelectorAll('.medium-editor-toolbar').length).toBe(1);
         });
 
+        it('should set custom className to the editor toolbar', function () {
+            expect(document.querySelectorAll('.medium-editor-toolbar').length).toBe(0);
+            var editor = this.newMediumEditor('.editor', {
+                    toolbar: {
+                        withClassName: 'my-custom-classname'
+                    }
+                }),
+                toolbar = editor.toolbar.getToolbarElement();
```

Comment:
Test failed, because select toolbar code.

Fix like this:
```
- toolbar = editor.toolbar.getToolbarElement();
+ toolbar = editor.getExtensionByName('toolbar').getToolbarElement();
```